packages/vm/src/cache.rs

```diff
@@ -132,6 +132,8 @@ pub struct AnalysisReport {
     pub entrypoints: BTreeSet<Entrypoint>,
     /// The set of capabilities the contract requires.
     pub required_capabilities: BTreeSet<String>,
+    /// The contract state version exported set by the contract developer
+    pub contract_state_version: Option<u64>,
```

Comment:
This unfortunately is a breaking change. We might get away with it (and make this struct non_exhaustive) though ... 🤫

Reply:
I think we *should* get away with it. I don't really think that there are many downstream users importing the VM package, and I especially don't think that they are exhaustively matching the `AnalysisReport` (or contructing their own).